x-pack/plugins/security_solution/server/lib/sourcerer/routes/index.ts

```diff
@@ -72,12 +72,15 @@ export const createSourcererDataViewRoute = (
         const siemDataViewTitle = siemDataView ? siemDataView.title.split(',').sort().join() : '';
         if (siemDataView == null) {
           try {
-            siemDataView = await dataViewService.createAndSave({
-              allowNoIndex: true,
-              id: dataViewId,
-              title: patternListAsTitle,
-              timeFieldName: DEFAULT_TIME_FIELD,
-            });
+            siemDataView = await dataViewService.createAndSave(
```

Comment:
Is it possible to add the unit test here?

Reply:
We can check that we are passing it, but we are not adding any functionality. the `createAndSave` method is tested in `dataViewService.createAndSave`. Like I said in the description, the only way to test the implications for security solution in a useful manner would be a functional test, and this is an edge case so I don't think it's wise to take up time functional testing something like this on every single build.

Reply:
Thank you for adding the test!